css/properties/overflow.json

```
@@ -63,12 +63,26 @@
               "edge": {
                 "version_added": false
               },
-              "firefox": {
-                "version_added": false
-              },
-              "firefox_android": {
-                "version_added": false
-              },
+              "firefox": [
+                {
+                  "version_added": "81"
+                },
+                {
+                  "alternative_name": "-moz-hidden-unscrollable",
+                  "version_added": null,
+                  "version_removed": "81"
+                }
+              ],
+              "firefox_android": [
+                {
+                  "version_added": "81"
+                },
+                {
+                  "alternative_name": "-moz-hidden-unscrollable",
+                  "version_added": null,
```

Firefox for Android's first release:

```suggestion
                  "version_added": "4",
```